src/js/views/maps/LayerCategoryListView.js

```diff
@@ -47,18 +53,30 @@ define(
 
         /**
         * The array of layer categories to display in the list
-        * @type {LayerCategoryItemView[]}
+        * @type {ExpansionPanel[]}
         */
-        layerCategoryItemViews: undefined,
+        panels: undefined,
 
         /**
         * Executed when a new LayerCategoryListView is created
         * @param {Object} options - A literal object with options to pass to the view
         */
         initialize(options) {
           if (options.collection instanceof AssetCategories) {
-            this.layerCategoryItemViews = options.collection.map(categoryModel => {
-              return new LayerCategoryItemView({model: categoryModel});
+            this.panelsModel = new ExpansionPanelsModel({ isMulti: true });
```

Comment:
does this need to be an instance variable?

Reply:
It does not, making local